ripc-core/src/main/java/io/ripc/internal/Publishers.java

```diff
@@ -0,0 +1,48 @@
+package io.ripc.internal;
+
+import org.reactivestreams.Publisher;
+import org.reactivestreams.Subscriber;
+import org.reactivestreams.Subscription;
+
+/**
+ * Temporary utility class for creating and transforming {@link Publisher}s.
+ */
+public class Publishers {
+
+    public static <T> Publisher<T> just(final T value) {
+        return new Publisher<T>() {
+            @Override
+            public void subscribe(final Subscriber<? super T> s) {
+                s.onSubscribe(new Subscription() {
+                    @Override
+                    public void request(long n) {
+                        s.onNext(value);
```

Comment:
Sure, just to be clear, the implementation is not feature complete. It was a strawman to start discussion in light of all layers being implemented.

>> The Specifications class was intended to be the helper that aggregated these checks.

Not sure if having an independent Specifications class helps in this case. How does it makes sure that everyone abides by the spec?